test/apps/network/tcp_poll_err.c

```diff
@@ -0,0 +1,263 @@
+// SPDX-License-Identifier: MPL-2.0
+
+#define _GNU_SOURCE
+
+#include <unistd.h>
+#include <sys/signal.h>
+#include <sys/socket.h>
+#include <sys/poll.h>
+#include <netinet/in.h>
+#include <arpa/inet.h>
+#include <fcntl.h>
+#include <stddef.h>
+
+#include "test.h"
+
+#define S_PORT htons(0x1238)
+
+static struct sockaddr_in sk_addr;
+int sk_listen;
+int sk_connect;
+int sk_accept;
+struct pollfd pfd = { .events = POLLIN | POLLOUT | POLLRDHUP };
+char buf[4096] = { 'a' };
```

Comment:
Well, even though they are just tests. But they seem a bit too noisy to me. Some suggestions:
 - Why is only `sk_addr` static? We should either mark all variables as `static` or remove the `static` keyword from `sk_addr`.
 - `sk_listen`/`sk_connect`/`sk_accept` are obviously similar things. They should have empty lines to isolate them from other unrelated things.